tests/tests/smt_tests.cpp

```diff
@@ -542,14 +591,6 @@ BOOST_AUTO_TEST_CASE( setup_apply )
    FC_LOG_AND_RETHROW()
 }
 
-/*
- * SMT legacy tests
- *
- * The logic tests in legacy tests *should* be entirely duplicated in smt_operation_tests.cpp
- * We are keeping these tests around to provide an additional layer re-assurance for the time being
- */
-FC_TODO( "Remove SMT legacy tests and ensure code coverage is not reduced" );
```

Comment:
Do we no longer need to remove these tests or is there a compelling reason to keep them?